src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/modifier/interfacememberimpliedmodifier/Example1.java

```diff
@@ -0,0 +1,38 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="InterfaceMemberImpliedModifier"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.modifier.interfacememberimpliedmodifier;
+
+import java.util.List;
+
+// xdoc section -- start
+class Example1 {
+  public interface AddressFactory {
+
+    public static final String UNKNOWN = "Unknown";
+    // violation below, 'Implied modifier 'final' should be explicit'
+    String OTHER = "Other";
+    // violation above, 'Implied modifier 'public' should be explicit'
+    // violation 2 lines above 'Implied modifier 'static' should be explicit'
```

Comment:
Please use `// 2 violations above:`
It looks more clearly

Reply:
Done.